evalml/automl/automl_search.py

```diff
@@ -174,6 +178,12 @@ def __init__(self,
 
             pipeline_parameters (dict): A dict of the parameters used to initalize a pipeline with.
 
+            sampler_method (str): The data sampling component to use in the pipelines if the problem type is classification and the target balance is smaller than the sampler_balanced_ratio.
+                Either 'auto', which will use our preferred sampler for the data, the name of the sampling component to use, or None. Defaults to 'auto'.
+
+            sampler_balanced_ratio (float): The minority:majority class ratio that we consider balanced, so a 1:4 ratio would be equal to 0.25. If the class balance is larger than this provided value,
```

Comment:
@bchen1116 I see this doesn't get passed down to `IterativeAlgorithm`. What's the intended behavior of this parameter? I would assume that setting a value here controls how much sampling is done during search but as things are now, I believe each undersampler would get init with the default value regardless of the value of this parameter.

Reply:
Ah good call! I thought I had this but I must've deleted it at some point...  will fix!